sdk/storage/Azure.Storage.Common/tests/Shared/TestConfigurations.cs

```diff
@@ -122,6 +128,12 @@ public class TestConfigurations
         public static TenantConfiguration DefaultTargetManagedDiskTenant =>
             GetTenant("TargetManagedDiskTenant", s_configurations.Value.TargetManagedDiskTenantName);
 
+        /// <summary>
+        /// Gets a tenant to use for any tests related to blob or container soft delete.
+        /// </summary>
+        public static TenantConfiguration DefaultTargetSoftDeleteTenant =>
+            GetTenant("TargetBlobAndContainerSoftDeleteTenant", s_configurations.Value.TargetSoftDeleteTenantName);
```

Comment:
Is Soft Delete an account level setting that applies regardless of whether you're using Blobs/Files?

Reply:
No, there is Container Soft Delete and Blob Soft Delete for the Blobs service, and Share Soft Delete for Files.  I'd like to just make 1 new test account with all soft delete features enabled.

The reason I want a separate account is that 1. it takes time to enable/disable these features on an account, and 2. if you list with the "Deleted" option, and soft delete has been enabled for weeks, the resulting list could be huge.

Reply:
Got it - I'm on board with the separate account.  Just wonder if we should call the tenant name `"TargetBlobsAndFilesSoftDeleteTenant"` or just `"TargetSoftDeleteTenant"` since you need to enable it for both services?  I don't feel very strongly here.

Reply:
I'll add additional documentation around what settings this tenant requres.